### PR TITLE
ft(new seeds): add new user seeders

### PR DESCRIPTION
**Pull request by rwamugema**
### PR Description
Add new user seeds.
### Description of tasks that were expected to be completed

- [x] Add new seeders in the user seeds

### Functionality
Add new user information in the user seeds 
### How has this been tested?
Login into one organizations with email of the new users added in the seed file.
### PR Checklist:
- [x] Add new seeders.
### Track PR
Trello Link https://trello.com/c/3SSuXyvr/66-add-new-seeds
### Screenshots (If appropriate)
None

### CODE REVIEW

File: src/resolvers/attendance.resolvers.ts
Comment (nshutip) [inline src/resolvers/attendance.resolvers.ts:1]: Why were lines 60 - 62 commented out?

File: src/seeders/users.seed.ts
Comment (nshutip) [inline src/seeders/users.seed.ts:1]: Why were the user role "trainee" changed to "user"??

File: Dockerfile
Comment (nshutip) [inline Dockerfile:1]: Was the docker file supposed to be updated while adding new user seeders???

**Review by nshutip — COMMENTED** [review]
There are many changes made that are not relevant to the feauture to be added by this PR.

**Review by Lydie260 — APPROVED** [review]

**Review by rukundo-kevin — APPROVED** [review]

File: src/resolvers/attendance.resolvers.ts
Comment (rwamugema) [reply src/resolvers/attendance.resolvers.ts:1]: It was because I haven't rebased yet but now it is reflecting only to my changes.

**Review by nshutip — APPROVED** [review]

File: src/seeders/users.seed.ts [hunk 1]
```diff
@@ -10,10 +10,22 @@ const seedUsers = async () => {
...
+      lastName: 'Lydia',
+      email: 'gatarelydie370@gmail.com',
+      guthubUserName: '',
+    },
+    {
+      firstName: ' Mystica Dukuzeyezu',
+      lastName: 'Marie Rose',
+      email: 'dukumystica20@gmail.com',
+      guthubUserName: '',
     },
```
Comment (ceelogre) [inline src/seeders/users.seed.ts]: Remove this object. She will be leaving soon.

File: src/seeders/users.seed.ts [hunk 2]
```diff
@@ -22,41 +34,29 @@ const seedUsers = async () => {
...
+      firstName: 'Ndayambaje',
+      lastName: 'Virgile',
+      email: 'ndayambajevgschooling@gmail.com',
+      guthubUserName: '',
     },
     {
       firstName: 'Benn Dalton',
       lastName: 'IRADUKUNDA',
       email: 'irabd44@gmail.com',
       guthubUserName: 'blackd44',
```
Comment (ceelogre) [inline src/seeders/users.seed.ts]: Should be removed.

File: src/seeders/users.seed.ts [hunk 3]
```diff
@@ -70,22 +70,9 @@ const seedUsers = async () => {
...
-      firstName: 'Eric',
-      lastName: 'Ndungutse',
-      email: 'dav.ndungutse@gmail.com',
-      guthubUserName: 'ericndungutse',
+      firstName: 'Ntwari',
+      lastName: 'Charles',
+      email: 'ntwarichar@gmail.com',
     },
     {
       firstName: 'Eric',
```
Comment (ceelogre) [inline src/seeders/users.seed.ts]: Which account is this?

**Review by ceelogre — CHANGES_REQUESTED** [review]

**Review by ceelogre — APPROVED** [review]

File: src/seeders/users.seed.ts [hunk 4]
```diff
@@ -22,41 +34,29 @@ const seedUsers = async () => {
       guthubUserName: 'Marthatwesi',
```
Comment (ceelogre) [inline src/seeders/users.seed.ts:34]: This should be removed too.

**Pull request merged**